db/db_bloom_filter_test.cc

```diff
@@ -1343,8 +1344,9 @@ TEST_F(DBBloomFilterTest, OptimizeFiltersForHits) {
   for (int i = 0; i < numkeys; i += 2) {
     keys.push_back(i);
   }
-  std::random_shuffle(std::begin(keys), std::end(keys));
-
+  std::random_device rng;
```

Comment:
I think we should unify all of these, including

test_util/transaction_test_util.cc:  std::shuffle(set_vec.begin(), set_vec.end(), std::random_device{});

with a single implementation in util/random.h,cc

Though we can do this after merging this PR.

Reply:
@pdillinger Yes you are right, I would be happy to propose something after this PR if it's good for you. Else I can try here, but I would prefer making it in another step.